ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -61,19 +61,19 @@ azure_connection::list_objects(
         blob_parameters.PageSizeHint = 1;
 
     try {
-        _azure_client.ListBlobs(blob_parameters);
+        auto list_blobs_response = _azure_client.ListBlobs(blob_parameters);
+        do {
+            for (const auto blob_item : list_blobs_response.Blobs)
```

Comment:
This should probably be `const auto& blob_item`, unless the `substr` method is somehow non-const.

Reply:
[c7dc520](https://github.com/wiredtiger/wiredtiger/pull/8984/commits/c7dc52029a313d3bbf1f7a35021935137612caa3)